client/state/ui/action-log/reducer.js

```diff
@@ -16,7 +16,12 @@ import {
 	SITE_SETTINGS_RECEIVE,
 } from 'state/action-types';
 
+const relevantAnalyticsEvents = [
+	'calypso_themeshowcase_theme_click',
+];
+
 const relevantTypes = {
+	ANALYTICS_EVENT_RECORD: isRelevantAnalytics,
```

Comment:
I'd say this hints at an underlying structural issue: things of semantically different types are stored in a common structure, tho they require rather different [treatment](https://github.com/Automattic/wp-calypso/pull/12438/files#diff-b01f7a026a0a958667df53ee8b7e6c29R37). What about having a separate `relevantTypesPredicates` object dedicated to functions like `isRelevantAnalytics`? (`relevantTypes` could then be turned into an array.)